<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>770</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_28">
    <item row="0" column="0" rowspan="3">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>UDP连接与设备初始化</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="2" column="3">
            <widget class="QLineEdit" name="lineEdit_XWJ_Port"/>
           </item>
           <item row="1" column="2">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>主机端口：</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0" colspan="2">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>设备ID：</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="3" column="0" colspan="4">
            <widget class="QPushButton" name="pushButton_Init">
             <property name="text">
              <string>初始化设备UDP</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>下位机IP：</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="lineEdit_XWJ_IP"/>
           </item>
           <item row="1" column="3">
            <widget class="QLineEdit" name="lineEdit_ZJ_Port"/>
           </item>
           <item row="4" column="0" colspan="4">
            <widget class="QPushButton" name="pushButton_CMD_CX">
             <property name="text">
              <string>发送查询命令</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>主机IP：</string>
             </property>
            </widget>
           </item>
           <item row="5" column="0" colspan="4">
            <widget class="QPushButton" name="pushButton_CMD_DeviceInfo">
             <property name="text">
              <string>加载设备信息</string>
             </property>
            </widget>
           </item>
           <item row="2" column="2">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>下位机端口：</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="lineEdit_ZJ_IP"/>
           </item>
           <item row="0" column="2" colspan="2">
            <widget class="QComboBox" name="comboBox_DeviceID">
             <item>
              <property name="text">
               <string>1</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>2</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>3</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>4</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>5</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>6</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>7</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>8</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>9</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>10</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="6" column="0">
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>设备信息：</string>
             </property>
            </widget>
           </item>
           <item row="6" column="1" colspan="3">
            <widget class="QTextEdit" name="textEdit_DeviceInfo">
             <property name="font">
              <font>
               <pointsize>9</pointsize>
              </font>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <layout class="QGridLayout" name="gridLayout_10">
      <item row="0" column="0">
       <widget class="QGroupBox" name="groupBox_4">
        <property name="title">
         <string>设置参数</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_12">
         <item row="0" column="0">
          <layout class="QGridLayout" name="gridLayout_11">
           <item row="5" column="1" colspan="2">
            <widget class="QLineEdit" name="lineEdit_Param_CFDelay"/>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label_11">
             <property name="text">
              <string>触发间隔：</string>
             </property>
            </widget>
           </item>
           <item row="2" column="2">
            <widget class="QCheckBox" name="checkBox_Param_SC_DOWM">
             <property name="text">
              <string>下降沿</string>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">buttonGroup_2</string>
             </attribute>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QCheckBox" name="checkBox_Param_SR_UP">
             <property name="text">
              <string>上升沿</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">buttonGroup</string>
             </attribute>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>输出触发：</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>灰度等级（1-16）：</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QCheckBox" name="checkBox_Param_SC_UP">
             <property name="text">
              <string>上升沿</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">buttonGroup_2</string>
             </attribute>
            </widget>
           </item>
           <item row="4" column="1" colspan="2">
            <widget class="QLineEdit" name="lineEdit_Param_CFPicnum"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>输入触发：</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1" colspan="2">
            <widget class="QLineEdit" name="lineEdit_Param_Gray"/>
           </item>
           <item row="0" column="1" colspan="2">
            <widget class="QLineEdit" name="lineEdit_Param_Delay"/>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>图片频率（Hz）：</string>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="label_12">
             <property name="text">
              <string>触发延时：</string>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QCheckBox" name="checkBox_Param_SR_DOWM">
             <property name="text">
              <string>下降沿</string>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">buttonGroup</string>
             </attribute>
            </widget>
           </item>
           <item row="6" column="0" colspan="3">
            <widget class="QPushButton" name="pushButton_CMD_SetParam">
             <property name="text">
              <string>发送设置参数命令</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="2" colspan="2">
     <layout class="QGridLayout" name="gridLayout_13">
      <item row="0" column="0">
       <widget class="QGroupBox" name="groupBox_5">
        <property name="title">
         <string>加载图像功能</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_14">
         <item row="0" column="0">
          <layout class="QGridLayout" name="gridLayout_15">
           <item row="0" column="2">
            <widget class="QPushButton" name="pushButton_ChooseBin">
             <property name="text">
              <string>选择Bin文件</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QPushButton" name="pushButton_SendTwo">
             <property name="text">
              <string>发送二值</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1" colspan="2">
            <widget class="QLineEdit" name="lineEdit_SendPicnum"/>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_13">
             <property name="text">
              <string>图片起始加载位置</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1" colspan="2">
            <widget class="QLineEdit" name="lineEdit_SendPicStarP"/>
           </item>
           <item row="4" column="1">
            <widget class="QPushButton" name="pushButton_SendEight">
             <property name="text">
              <string>发送灰度</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_14">
             <property name="text">
              <string>文件夹内图片数量：</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1" colspan="2">
            <widget class="QLineEdit" name="lineEdit_Picnum_Dir">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QPushButton" name="pushButton_ChooseEightPic">
             <property name="text">
              <string>选择八位灰度文件夹</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QPushButton" name="pushButton_ChooseTwoPic">
             <property name="text">
              <string>选择二值图像文件夹</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_15">
             <property name="text">
              <string>发送图片数量(张数)</string>
             </property>
            </widget>
           </item>
           <item row="6" column="0">
            <widget class="QLabel" name="label_16">
             <property name="text">
              <string>发送数组Demo</string>
             </property>
            </widget>
           </item>
           <item row="4" column="2">
            <widget class="QPushButton" name="pushButton_SendBin">
             <property name="text">
              <string>发送Bin</string>
             </property>
            </widget>
           </item>
           <item row="6" column="1" colspan="2">
            <widget class="QPushButton" name="pushButton_SendChar">
             <property name="text">
              <string>发送数组</string>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="label_18">
             <property name="text">
              <string>数组Demo-图片宽和高：</string>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <widget class="QLineEdit" name="lineEdit_Width"/>
           </item>
           <item row="5" column="2">
            <widget class="QLineEdit" name="lineEdit_Height"/>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="1" rowspan="2">
     <layout class="QGridLayout" name="gridLayout_19">
      <item row="0" column="0">
       <widget class="QGroupBox" name="groupBox_7">
        <property name="title">
         <string>播放操作</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_26">
         <item row="0" column="0">
          <layout class="QGridLayout" name="gridLayout_20">
           <item row="1" column="0" colspan="2">
            <widget class="QComboBox" name="comboBox_CMD_Play">
             <item>
              <property name="text">
               <string>内部单次</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>内部循环</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>外部单次</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>外部循环</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>可变序列内部单次</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>可变序列内部循环</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>可变序列外部单次</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>可变序列外部循环</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_19">
             <property name="text">
              <string>图片张数</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_17">
             <property name="text">
              <string>开始位置</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="lineEdit_PlayOffset"/>
           </item>
           <item row="3" column="1">
            <widget class="QLineEdit" name="lineEdit_PicCount"/>
           </item>
           <item row="5" column="0">
            <widget class="QPushButton" name="pushButton_CMD_PlayStop">
             <property name="text">
              <string>暂停</string>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <widget class="QPushButton" name="pushButton_CMD_Stop">
             <property name="text">
              <string>中止</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0" colspan="2">
            <widget class="QPushButton" name="pushButton_CMD_Play">
             <property name="text">
              <string>播放</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="2" colspan="2">
     <layout class="QGridLayout" name="gridLayout_21">
      <item row="0" column="0">
       <widget class="QGroupBox" name="groupBox_8">
        <property name="title">
         <string>其他操作</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_23">
         <item row="0" column="0">
          <layout class="QGridLayout" name="gridLayout_22">
           <item row="0" column="0">
            <widget class="QPushButton" name="pushButton_CMD_UPDOWM">
             <property name="text">
              <string>上下翻转</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QPushButton" name="pushButton_CMD_DateReverse">
             <property name="text">
              <string>数据反向</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QPushButton" name="pushButton_CMD_SoftCF">
             <property name="text">
              <string>软件触发</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QPushButton" name="pushButton_CMD_DMDFloat">
             <property name="text">
              <string>DMD-Float</string>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QPushButton" name="pushButton_CMD_Reset">
             <property name="text">
              <string>DMD-复位</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="3" rowspan="2">
     <layout class="QGridLayout" name="gridLayout_24">
      <item row="0" column="0">
       <widget class="QGroupBox" name="groupBox_9">
        <property name="title">
         <string>自定义命令</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_27">
         <item row="0" column="0">
          <layout class="QGridLayout" name="gridLayout_25">
           <item row="0" column="0">
            <widget class="QTextEdit" name="textEdit_ZDY"/>
           </item>
           <item row="1" column="0">
            <widget class="QPushButton" name="pushButton_SendZDYCmd">
             <property name="text">
              <string>发送自定义命令</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="0" rowspan="2">
     <layout class="QGridLayout" name="gridLayout_16">
      <item row="0" column="0">
       <widget class="QGroupBox" name="groupBox_6">
        <property name="title">
         <string>设置序列</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_18">
         <item row="0" column="0">
          <layout class="QGridLayout" name="gridLayout_17">
           <item row="0" column="0">
            <widget class="QTableWidget" name="tableWidget"/>
           </item>
           <item row="1" column="0">
            <widget class="QPushButton" name="pushButton_CMD_SetSequence">
             <property name="text">
              <string>设置序列</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="1" rowspan="2" colspan="2">
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>用户操作与操作结果</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_6">
         <item row="0" column="0">
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="0" column="0">
            <widget class="QPushButton" name="pushButton_User_clear">
             <property name="text">
              <string>清除</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QTextEdit" name="textEdit_User">
             <property name="font">
              <font>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item row="4" column="3">
     <layout class="QGridLayout" name="gridLayout_7">
      <item row="0" column="0">
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string>下位机反馈信息</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_9">
         <item row="0" column="0">
          <layout class="QGridLayout" name="gridLayout_8">
           <item row="0" column="0">
            <widget class="QPushButton" name="pushButton_FeedBack_clear">
             <property name="text">
              <string>清除</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QTextEdit" name="textEdit_FeedBack">
             <property name="font">
              <font>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
  <buttongroup name="buttonGroup_2"/>
 </buttongroups>
</ui>
